content/swift/concepts/protocols/terms/decoder/decoder.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'Decoder'
+Description: 'A Swift protocol that converts external data into Swift types.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Swift'
```

Comment:
```suggestion
```
please choose the appropriate tags from the official [documentation](https://github.com/Codecademy/docs/blob/main/documentation/tags.md). Swift is not listed in it.